PHP 5.4 strict standards errors

EdenwareApps opened the pull request:
I get some annoying strict standards errors with PHP 5.4.27. Fixed that just by adding "static" before those functions.

Strict Standards: Non-static method Pommo_Helper_L10n::init() should not
be called statically in .../classes/Pommo.php on line 179

Strict Standards: Non-static method Pommo_Helper_L10n::_setlocale()
should not be called statically in .../classes/Pommo_Helper_L10n.php on
line 37

Strict Standards: Non-static method Pommo_Helper_L10n::translate()
should not be called statically in .../classes/Pommo.php on line 368

Strict Standards: Non-static method Pommo_Pending::getBySubID() should
not be called statically in .../pending.php on line 41

Strict Standards: Non-static method Pommo_Pending::makeDB() should not
be called statically in .../classes/Pommo_Pending.php on line 148

Strict Standards: Non-static method Pommo_Type::pending() should not be
called statically in .../classes/Pommo_Pending.php on line 46

Strict Standards: Non-static method Pommo_Mailing::getSubject() should not be called statically in .../ajax/history.rpc.php on line 57

Strict Standards: Non-static method Pommo_Mailing::getNotices() should not be called statically in .../ajax/history.rpc.php on line 58

EdenwareApps committed:
PHP 5.4 strict standards errors

Just added "static" before some functions.

Strict Standards: Non-static method Pommo_Helper_L10n::init() should not
be called statically in .../classes/Pommo.php on line 179

Strict Standards: Non-static method Pommo_Helper_L10n::_setlocale()
should not be called statically in .../classes/Pommo_Helper_L10n.php on
line 37

Strict Standards: Non-static method Pommo_Helper_L10n::translate()
should not be called statically in .../classes/Pommo.php on line 368

Strict Standards: Non-static method Pommo_Pending::getBySubID() should
not be called statically in .../pending.php on line 41

Strict Standards: Non-static method Pommo_Pending::makeDB() should not
be called statically in .../classes/Pommo_Pending.php on line 148

Strict Standards: Non-static method Pommo_Type::pending() should not be
called statically in .../classes/Pommo_Pending.php on line 46

EdenwareApps committed:
PHP 5.4 more strict standards errors

Just added "static" before some more functions.

soonick commented:
Thanks @efoxbr,

Can you please squash both commits and re-send the pull request?

Thanks,
Adrian

EdenwareApps committed:
PHP 5.4 strict standards errors

I get some annoying strict standards errors with PHP 5.4.27. Fixed that
just by adding "static" before those functions.

Strict Standards: Non-static method Pommo_Helper_L10n::init() should not
be called statically in .../classes/Pommo.php on line 179

Strict Standards: Non-static method Pommo_Helper_L10n::_setlocale()
should not be called statically in .../classes/Pommo_Helper_L10n.php on
line 37

Strict Standards: Non-static method Pommo_Helper_L10n::translate()
should not be called statically in .../classes/Pommo.php on line 368

Strict Standards: Non-static method Pommo_Pending::getBySubID() should
not be called statically in .../pending.php on line 41

Strict Standards: Non-static method Pommo_Pending::makeDB() should not
be called statically in .../classes/Pommo_Pending.php on line 148

Strict Standards: Non-static method Pommo_Type::pending() should not be
called statically in .../classes/Pommo_Pending.php on line 46

Strict Standards: Non-static method Pommo_Mailing::getSubject() should
not be called statically in .../ajax/history.rpc.php on line 57

Strict Standards: Non-static method Pommo_Mailing::getNotices() should
not be called statically in .../ajax/history.rpc.php on line 58

EdenwareApps committed:
Merge branch 'master' of https://github.com/efoxbr/poMMo

Conflicts:
	classes/Pommo_Helper_L10n.php
	classes/Pommo_Mailing.php
	classes/Pommo_Pending.php
	classes/Pommo_Type.php
